e2e/react_native_0_61x/__tests__/init.test.ts

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs';
+import path from 'path';
+import mkdirp from 'mkdirp';
+import { run, cleanup, yarnCommand } from '../../utils/common';
+import { runHaul } from '../../utils/runHaul';
+
+const TEST_PROJECT_DIR = path.resolve(__dirname, '__fixtures__/testapp');
+
+const CONFIG_FILE_PATH = path.resolve(TEST_PROJECT_DIR, 'haul.config.js');
+const BABEL_CONFIG_FILE_PATH = path.resolve(
+  TEST_PROJECT_DIR,
+  'babel.config.js'
+);
+const XCODE_PROJECT_PATH = path.resolve(
+  TEST_PROJECT_DIR,
+  'ios/testapp.xcodeproj/project.pbxproj'
+);
+const GRADLE_PATH = path.resolve(TEST_PROJECT_DIR, 'android/app/build.gradle');
+const ENTER_KEY = '\x0d';
+
+beforeEach(() => {
+  cleanup(path.join(TEST_PROJECT_DIR, '..'));
+  mkdirp.sync(path.join(TEST_PROJECT_DIR, '..'));
+});
+afterEach(() => cleanup(path.join(TEST_PROJECT_DIR, '..')));
+
+test('init command on a fresh react-native 0.61 project', done => {
+  run(`${yarnCommand} init -y`, path.join(TEST_PROJECT_DIR, '..'));
+  run(`${yarnCommand} add react-native-cli`, path.join(TEST_PROJECT_DIR, '..'));
+  run(
+    `${yarnCommand} react-native init testapp --version 0.61.0`,
```

Comment:
We advise using `npx` now, which uses new `init`. So essentially a single command, without yarn init, would suffice:
```
npx react-native init testapp --version 0.61.0
```